jobs/integr8ly/pds-install.yaml

```diff
@@ -162,6 +188,27 @@
                                     """
                                 }
                             }
+                            if (INSTALL_BACKUPS.toString() == 'true') {
+                                String backupsNamespace = 'openshift-integreatly-backups';
```

Comment:
Is this always with `openshift-` prefix ?

Reply:
yes

Reply:
yep, I think we can have this hard-coded. There is a param for install playbook to change this value, however, effectively only this default value is used.